### DISCUSSION TITLE
how can i modify the kubelet-arg after k3s start

### DISCUSSION THREAD

**Discussion by wang-xiaowu** [k3s-io/k3s#5434] [Q&A] [answered]
excuse me, is there anyone can tell me how can i modify the kubelet-arg after k3s start

**Comment by brandond** [comment, marked as answer]
Add `--kubelet-arg=foo=bar` to the configuration in the k3s systemd unit, then reload the systemd configuration and restart k3s.

**Reply by wang-xiaowu** [reply]
thank u
